etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,437 @@
+---
+#########################################
+# Stackhpc default firewall configuration
+#########################################
+
+stackhpc_firewalld_zones: |
+  {% set network_zones = [] %}
+  {% for network in network_interfaces %}
+  {% if network | net_zone %}
+  {% set _ = network_zones.append({'zone': network | net_zone }) %}
+  {% endif %}
+  {% endfor %}
+  {{ network_zones }}
+
+# Rules are filtered to ensure that none are conflicting
+stackhpc_firewalld_rules: |
+  {% set stackhpc_firewalld_rules_filtered = [] %}
+  {% for rule in stackhpc_firewalld_rules_unfiltered %}
+  {% if rule | ansible.utils.remove_keys('state') not in stackhpc_firewalld_rules_filtered | map('ansible.utils.remove_keys', 'state') %}
+  {% set _ = stackhpc_firewalld_rules_filtered.append(rule) %}
+  {% endif %}
+  {% endfor %}
+  {{ stackhpc_firewalld_rules_filtered }}
+
+stackhpc_firewalld_rules_unfiltered: |
+  {{  (stackhpc_controller_firewalld_rules if 'controllers' in group_names else []) +
+      (stackhpc_compute_firewalld_rules if 'compute' in group_names else []) +
+      (stackhpc_storage_firewalld_rules if 'storage' in group_names else []) +
+      (stackhpc_monitoring_firewalld_rules if 'monitoring' in group_names else []) +
+      (stackhpc_seed_firewalld_rules if 'seed' in group_names else []) +
+      (stackhpc_seed_hypervisor_firewalld_rules if 'seed-hypervisor' in group_names else []) +
+      (stackhpc_wazuh_manager_infra_vm_firewalld_rules if 'wazuh-manager' in group_names else []) +
+      (stackhpc_ansible_control_infra_vm_firewalld_rules if 'ansible-control' in group_names else []) }}
```

Comment:
Need to document the ansible-control group? Or just localhost?

Reply:
Now using localhost